components/Slider/src/index.svelte

```diff
@@ -16,6 +16,7 @@
 	export let attrs: KSliderProps['attrs'] = {};
 	export let cls: KSliderProps['cls'] = undefined;
 	export let showTooltip: KSliderProps['showTooltip'] = true;
+	export let format: KSliderProps['format'] = '';
```

Comment:
Sorry, maybe my description of #342 is not accurate enough. It should actually be a function called format. The parameter is the current slider value and the return value is the user-defined formatted content.it might be something like this
```
format: (value: number) => number | string
```

Reply:
ok, I got it